chapters/equations.tex

```diff
@@ -803,7 +803,7 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \begin{nonnormative}
 In the case a parameter has both a binding equation and \lstinline!fixed = false! a diagnostics is recommended, but the parameter should be solved from the binding equation.
 
-Non-discrete (that is continuous-time) \lstinline!Real! variables \lstinline!vc! have exactly one initialization value since the rules above assure that during initialization
+Non-discrete \lstinline!Real! variables \lstinline!vc! have exactly one initialization value since the rules above assure that during initialization
```

Comment:
It seems inconsistent to skip the "-time" part here.  Was that for a reason, or should it simply be as below?
```suggestion
Non-discrete-time \lstinline!Real! variables \lstinline!vc! have exactly one initialization value since the rules above assure that during initialization
```